third_party/terraform/tests/resource_app_engine_service_split_traffic_test.go

```diff
@@ -0,0 +1,127 @@
+package google
+
+import (
+	"log"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
```

Comment:
```suggestion
	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
```

Reply:
This + the other helper import need to be fixed to use the new plugin SDK package.